sdk-base/src/main/java/com/mapbox/maps/plugin/gestures/generated/GesturesSettingsInterface.kt

```diff
@@ -97,6 +97,16 @@ interface GesturesSettingsInterface {
    * The amount by which the zoom level increases or decreases during a double-tap-to-zoom-in or double-touch-to-zoom-out gesture. 1.0 by default. Must be positive.
    */
   var zoomAnimationAmount: Float
+
+  /**
+   * Whether to adjust pitch to 0.0 automatically depending on zoom threshold when globe projection is used to improve overall gesture experience. Defaults to true.
```

Comment:
this line looks too long, can we add line breaker(s)?

Reply:
I'll check if it's supported in generators 👍